spikingjelly/activation_based/neuron/few_spike.py

```
@@ -0,0 +1,602 @@
+    def _run_table(
+        self,
+        gate: torch.Tensor,
+        theta: torch.Tensor,
+        h: torch.Tensor,
+        d: torch.Tensor,
+        return_sequence: bool,
+    ) -> torch.Tensor:
+        v = gate
+        y_seq = []
+        y = None
+        for theta_k, h_k, d_k in zip(theta.unbind(0), h.unbind(0), d.unbind(0)):
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)